vscode-sbt-scala/client/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "vscode-sbt-scala",
+  "displayName": "Scala (sbt)",
+  "version": "0.0.1",
+  "author": "Lightbend, Inc.",
+  "license": "BSD 3-Clause",
+  "publisher": "eed3si9n",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sbt/sbt"
+  },
+  "description": "VSCode part of the Scala language server",
+  "engines": {
+    "vscode": "^1.16.0"
+  },
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onLanguage:plaintext"
```

Comment:
That doesn't seem right.`"workspaceContains:build.sbt"` would be better, but unfortunately things tend to break when two language servers are active at the same time in vscode (see https://github.com/dragos/dragos-vscode-scala/pull/47). One solution would be to add code in the vscode client extension to not start the server if either `.ensime` or `.dotty-ide.json` ae present. (And then I can modify the dotty language server to start and talk to an instance of the sbt language server).

Reply:
Good catch. `"onLanguage:plaintext"` is a leftover from their sample code.